refaster-test-support/src/main/java/tech/picnic/errorprone/refaster/test/RefasterRuleCollection.java

```diff
@@ -173,6 +186,68 @@ private void reportIncorrectClassName(CompilationUnitTree tree, VisitorState sta
     }
   }
 
+  private void reportUnSortedElidedTypesAndStaticImports(
+      CompilationUnitTree compilationUnit, VisitorState state, SuggestedFix.Builder fixBuilder) {
+    new TreeScanner<@Nullable Void, @Nullable Void>() {
+      @Override
+      public @Nullable Void visitMethod(MethodTree tree, @Nullable Void unused) {
+        String methodName = tree.getName().toString();
+        if (ELIDED_TYPES_AND_STATIC_IMPORTS_METHOD_NAME.equals(methodName)) {
+          Optional<ReturnTree> returnTree =
+              tree.getBody().getStatements().stream()
+                  .filter(statement -> statement.getKind() == Kind.RETURN)
+                  .findFirst()
+                  .map(ReturnTree.class::cast);
+
+          returnTree
+              .map(ReturnTree::getExpression)
+              .map(MethodInvocationTree.class::cast)
+              .ifPresent(
+                  methodInvocationTree -> {
+                    List<? extends ExpressionTree> actualArgumentsOrdering =
+                        methodInvocationTree.getArguments();
+
+                    ImmutableList<? extends ExpressionTree> desiredArgumentOrdering =
+                        actualArgumentsOrdering.stream()
```

Comment:
Should we group argument types togther before sorting ?
i.e: Group & sort method invocation lexicographically > group & sort identifiers lexicographically, etc...

Reply:
> Do you have an example of the case you mean?

My thought was (As an example)
1. Group & sort Objects (Strings for eg).
2. Group & sort Class statemens (`*.class`).
3. Group & sort method arguments.
5. Group & sort primitives.

Essentially turning
```
ImmutableSet.of(List.class, "A", assertDoesNotThrow(() -> null), Iterables.class, "a", 1)
```
to 
```
ImmutableSet.of("A", "a", List.class, Iterables.class, assertDoesNotThrow(() -> null),1)
```